cyber-resilience-sig/coordination/cra-expert-group/foss-guidance-ope-source-hardware.md

```diff
@@ -0,0 +1,32 @@
+# Input to the CRA Expert Group's Open Source Workstrand on open source hardware
+
+## Context
+
+The European Commission sought the [Cyber Resilience SIG](../../)'s input to better understand some of the concerns raised by the open source harware commnunity as part of its consultation on the technical definitions of important and critical products. After consulting with members of the open source hardware community through the dedicated mailing list hosted by OpenForum Europe and some back and forth with the Commission, we agreed that the best course of action was to provide input to the Commission's Guidance on open source via the [CRA Expert Group](../../#cra-expert-group). This input would clarify what open source hardware is, demonstrate its functional equivalence to open source software, and argue that it should therefore be treated as open source software under the CRA.
+
+This input will be provided to the open source workstrand of the [CRA Expert Group](../../#cra-expert-group) at the very latest on June 16, 2025.
+
+## Draft submission
+
+> ### Open Source Hardware
+> 
+> The purpose of this section is to clarify how the CRA applies to open source hardware, particularly in relation to source files and their licensing.
+> 
+> Computer hardware is typically developed from source files—such as hardware description language (HDL) code, schematics, or layout files—which describe the structure and function of the final physical product. These files are analogous to source code in software development. The source files themselves are not products with digital elements under the CRA; they are design inputs, not functional hardware. The actual product only emerges when these source files are manufactured into physical components or integrated into a system.
+> 
+> When these source files are released under a free and open source licence, they are referred to as open source hardware. Like free and open source software, such open source hardware is often developed and shared collaboratively without commercial intent. In line with the treatment of FOSS, open source hardware source files should not, by themselves, trigger CRA obligations unless they are part of a commercial activity, for example monetised or embedded in a product.
```

Comment:
> ... unless they are part of a commercial activity ...

OpenHW Foundation activities explicitly allow for commercial activity. The majority of OpenHW members are commercial organizations that intend to integrate OpenHW IP into their commerical products.  Is this a problem for us?

Reply:
That said, we need to be careful in the wording here, to avoid implying that open source hardware is entirely non-commercial ("open source" does not mean "non-commercial). The text of the CRA was quite careful about this, for example:

> Article 3(14) ‘open-source software steward’ means a legal person, other than a manufacturer, that has the purpose or objective of systematically providing support on a sustained basis for the development of specific products with digital elements, qualifying as free and open-source software **and intended for commercial activities**, and that ensures the viability of those products;